Sources/SPTDataLoader/SPTDataLoaderRequestTaskHandler.m

```diff
@@ -179,7 +181,7 @@ - (NSURLSessionResponseDisposition)receiveResponse:(NSURLResponse *)response
 - (BOOL)mayRedirect
```

Comment:
this method name doesn't imply that it mutates anything, but it bumps `redirectCount`. out of scope for this PR to change it though, just thought i'd bring it up